app/assets/javascripts/mapknitter/Map.js

```diff
@@ -617,9 +617,81 @@ MapKnitter.Map = MapKnitter.Class.extend({
+      exportOpts: {
+        // exportUrl: 'http://34.74.118.242/api/v2/export/', // to swap to JS exporter
+        // exportStartUrl: 'http://34.74.118.242/api/v2/export/', // to swap to JS exporter
+        fetchStatusUrl: fetchStatusUrl
+      }
     }).addTo(map);
 
+    // customize the function that starts up the export
+    function fetchStatusUrl(opts) {
+      console.log('fetch status json', opts);
+
+      var scale = 0;
+      opts.collection.forEach(function(img) {
+        scale += img.cm_per_pixel;
+      });
+      // average of scales of each image
+      scale = parseInt(scale/opts.collection.length);
+
+      $.ajax({
+        url: 'https://export.mapknitter.org/export',
+        crossDomain: true,
+        type: 'POST',
+        data: {
+          collection: JSON.stringify(opts.collection),
+          scale: prompt("Choose a scale in 'centimeters per pixel' (where a smaller 50cm pixel is higher resolution - comparable to Google Maps - or a larger 200cm pixel is lower resolution):", scale) || opts.scale,
+          upload: true,
+        },
+        success: handleStatusResponse
+      });
+      // show exports
+      $('.export-tab').click();
+      $('.exports-tab').click();
+      window.location.hash = "#cloud-exports";
+    }
+
+    // receives the URL of status.json, and starts running the updater to repeatedly fetch from status.json;
+    // this may be overridden to integrate with any UI
+    function handleStatusResponse(status_url, opts) {
+      alert("Export has begun: leave this window open to be notified of completion. Due to a known issue, please refresh the page if you'd like to start another."); // https://github.com/publiclab/Leaflet.DistortableImage/issues/522
+      // this is for the JS exporter:
+      // var statusUrl = data.split('please visit, ')[1];
+      
+      // save the location of the status URL
+      $.ajax({	
+        url: "/export",	
+        type: 'POST',	
+        data: { 
+          status_url: 'https://export.mapknitter.org' + status_url,
+          map_id: mapknitter.map_id
+        }
+      }).done(function (data) {	
+        console.log('saved status.json URL to MapKnitter', data);	
+      });
+
+      // repeatedly fetch the status.json
+      var updateInterval = setInterval(function intervalUpdater() {
+        $.ajax('https://export.mapknitter.org/' + status_url + '?' + Date.now(), { // bust cache with timestamp
+          type: 'GET',
+          crossDomain: true,
+        }).done(function(data) {
+          // update the progress bar or spinner
+          // opts.updater(data);
+          data = JSON.parse(data);
+          console.log(data, data.status, data.jpg);
+          if (data && data.status == "complete") {
+            alert("Export completed at " + data.cm_per_pixel + " cm/px. JPG available at https://mapknitter-exports-warps.storage.googleapis.com/" + data.jpg.split('public/warps')[1] + " -- Please refresh page to view completed exports.");
```

Comment:
Here I set the alert to show the JPG and direct towards a reload. then cleared the interval. Wooo!